draft-issuance.md

```diff
@@ -0,0 +1,165 @@
+```
+ZIP: 
+Title: Smooth Out The Block Subsidy Issuance
+Owners: Jason McGee <jason@shieldedlabs.com>
+        Mark Henderson <mark@equilibrium.co>
+        Tomek Piotrowski <tomek@eiger.co>
+        Mariusz Pilarek <mariusz@eiger.co>
+Original-Authors: Nathan Wilcox
+Credits:
+Status: Draft
+Category: Consensus
+Created: 2023-08-23
+License: BSD-2-Clause
+```
+
+# Terminology
+
+The key words “MUST”, “SHOULD”, “SHOULD NOT”, “MAY”, “RECOMMENDED”, “OPTIONAL”,
+and “REQUIRED” in this document are to be interpreted as described in RFC 2119. [1]
+
+"Network upgrade" - to be interpreted as described in ZIP 200. [2]
+
+“Block Subsidy” - to be interpreted as described in the Zcash Protocol Specification (TODO ZIP Editors: link from comment).
+
+“Issuance” - the sum of Block Subsidies over time. (TODO ZIP Editors: work out if this definition is correct or can be removed).
+
+“`ZsfBalanceAfter(h)`” is the total ZEC available in the Zcash Sustainability Fund (ZSF) after the transactions
+in block `h`, described in ZIP draft-zsf.md. In this ZIP, the Sustainability Fund is used to pay out Block Subsidies
+from unmined ZEC, and other fund deposits.
+
+Let `PostBlossomHalvingInterval` be as defined in [#protocol-diffadjustment]_.
+
+
+# Abstract
+
+This ZIP proposes a change to how nodes calculate the block subsidy.
+
+Instead of following a step function around the 4-year halving intervals inherited
+from Bitcoin, we propose a slow exponential “smoothing” of the curve. The new issuance
+scheme would approximate the current issuance over 4-year intervals.
+
```

Comment:
In the abstract, it would be helpful to say that this ZIP depends on https://github.com/zcash/zips/pull/703

(This is already implied, but it helps to be specific.)

Reply:
done